snapd-lib/assertions/ubuntu-core-24-amd64.model

```diff
@@ -1,48 +1,49 @@
 type: model
```

Comment:
Had copied the stable model instead of dangerous - this is now tested better and works